cmd/kubebuilder/main.go

```diff
@@ -49,14 +48,15 @@ func main() {
 			"\nCurrent GOPATH=%s.  \nCurrent directory=%s", gopath, wd)
 	}
 	util.Repo = strings.Replace(wd, util.GoSrc+string(filepath.Separator), "", 1)
+
+	// add init command
 	initproject.AddInit(cmd)
+
+	// add version command
 	version.AddVersion(cmd)
 
-	if util.IsNewVersion() || util.IsProjectNotInitialized() {
-		v1.AddCmds(cmd)
-	} else {
-		v0.AddCmds(cmd)
-	}
+	// add all commands corresponding to v1 project
+	v1.AddCmds(cmd)
```

Comment:
This v1 package can be removed after v0 cleanup.

Reply:
Yes. I am keeping code deletion for next PR.